artiq/dashboard/waveform.py

```diff
@@ -85,17 +80,28 @@ def __init__(self):
         self.client = AsyncioClient()
 
     async def trigger_proxy_task(self):
-        if self.client.get_rpc_id()[0] is None:
-            raise AttributeError("Unable to identify RPC target. Is analyzer proxy connected?")
-        await self.client.trigger()
+        try:
+            if self.client.get_rpc_id()[0] is None:
+                raise IOError("no analyzer proxy configured")
```

Comment:
Changed to "missing connection to analyzer proxy control"

Reply:
IIRC it was added since it could be a common mistake to press the button without a connected proxy, and the sipyco client throws a somewhat ambiguous SyntaxError in this case, meaning it is not clear that the connection is missing

Reply:
Agreed, I'll remove this special case and make a PR in sipyco once this is pushed through.